.github/scripts/check-missing-readme-generation.sh

```diff
@@ -14,7 +14,7 @@ for file in $DOCIFY_FILES; do
   
   # Go to the directory and run cargo build
   cd "$DIR"
-  cargo check --features generate-readme || echo "Readme generation for $DIR failed. Ensure the crate compiles successfully and has a `generate-readme` feature which guards markdown compilation in the crate as follows: https://docs.rs/docify/latest/docify/macro.compile_markdown.html#conventions."
+  cargo check --features generate-readme || echo "Readme generation for $DIR failed. Ensure the crate compiles successfully and has a `generate-readme` feature which guards markdown compilation in the crate as follows: https://docs.rs/docify/latest/docify/macro.compile_markdown.html#conventions." && exit 1
```

Comment:
You would need to wrap around `{ echo .. && exit 1; }`, other the `&& exit 1` will be executed always.